libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -143,6 +142,7 @@ string IRGenerator::generate(
 	InternalDispatchMap internalDispatchMap = generateInternalDispatchFunctions();
 	t("functions", m_context.functionCollector().requestedFunctions());
 	t("subObjects", subObjectSources(m_context.subObjectsCreated()));
+	t("memoryInitCreation", memoryInit(!m_context.inlineAssemblySeen()));
```

Comment:
Maybe it is better to store this inside a local variable and add a comment: This has to be called after all other code generation is complete.